src/screens/AuthScreen/AuthScreen.tsx

```diff
@@ -37,77 +24,15 @@ const AuthScreen = () => {
   //TODO: add to constants
   const maxLength = 4;
   const handleSignIn = () => {
-    setGithubView(true);
+    // NOTE: toast until sign in with Github is implemented
+    Toast.show({
+      type: 'info',
+      text1: 'Sign in with GitHub coming soon...',
+      position: 'bottom',
+      bottomOffset: 80,
+    });
```

Comment:
> we already have a constant file, please move it to constant

This object should also come from constants

Reply:
sure !